src/app/hooks/useUser.ts

```diff
@@ -2,8 +2,14 @@ import { useEffect, useState } from 'react';
 import { Profile } from '../../types';
 import { getCurrentUser } from '../utils/user';
 
+const defaultUser: Profile = {
+  username: null,
+  displayName: null,
+  avatar: null,
+};
+
 export function useCurrentUser(): [Profile | null, string | null] {
-  const [currentUser, setProfile] = useState<Profile | null>(null);
+  const [currentUser, setProfile] = useState<Profile | null>(defaultUser);
```

Comment:
```suggestion
  const [currentUser, setCurrentUser] = useState<User>(defaultUser);
```
- `setProfile` -> `setCurrentUser`
- `Profile` -> `User`
- ` | null` -> ``

Reply:
Not reladed to this change, but I would reset the current user if `loadUser` fails.
Add to `catch`:
`setCurrentUser(defaultUser)`

Reply:
I was thinking, if the user is null, the defaults would be loaded anyway, but true its mor consistent